org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/WorkspaceManager.xtend

```diff
@@ -183,7 +183,7 @@ import java.util.HashSet
     }
     
     protected def boolean exists(URI uri) {
-        getProjectManager(uri)?.resourceSet.URIConverter.exists(uri, null)
+        getProjectManager(uri)?.resourceSet?.URIConverter.exists(uri, null)
```

Comment:
when getProjectManager(uri) returns null, resourceSet is null and getURIConverter is invoked on it. See the generated Java code for the potential null access.